### PR TITLE
WIP CliClient to winrm tar.gz distribution

### PR DESCRIPTION
**Pull request by bostko**
This allows to easily support powershell calls out of the box, just by editting the CliClient call.

I was thinking about some cli support. I found only one java cli which is an ant project.
When I have time I will include in this PR a dist module with shell script to start the CliClient.

For the Cli what I am about to do is:
define a syntax like:
cmd mycmd arg1 arg2 ...
ps My-ScriptLet arg1 arg2

For that I will use `org.apache.maven.shared.utils.cli.CommandLine.translateCommandline(String)` to parse arguments on each line.

### CODE REVIEW

**Comment by neykov** [comment]
That's definitely useful for testing. Keep in mind that any dependencies you add here will be pulled by projects using the library. Perhaps move this to `src/test/java`, so the command-line parser is a test dependency, not seen by the users of the library?

**Comment by bostko** [comment]
@neykov Of course I will make a dist project similar to brooklyn-dist

File: winrm4j/src/main/java/io/cloudsoft/winrm4j/winrm/CliClient.java [hunk 1]
```diff
@@ -23,18 +20,18 @@ public static void main(String[] args) throws Exception{
...
-        WinRmClient client = WinRmClient.builder(endpoint)
+        WinRmTool client = WinRmTool.Builder.builder(endpoint, username, password)
                 .disableCertificateChecks(true)
-                .credentials(username, password)
                 .workingDirectory("C:\\")
 //                .environment(env)
                 .build();
         int exitCode = 999;
         try {
-            exitCode = client.command(cmd, new PrintWriter(new OutputStreamWriter(System.out)), new PrintWriter(new OutputStreamWriter(System.err)));
```
Comment (neykov) [inline winrm4j/src/main/java/io/cloudsoft/winrm4j/winrm/CliClient.java]: This, at least in theory, allows for interleaving the stdout & stderr output, similar to how you'd see it in the console. Also the output will be visible as soon as the poll call completes, not at the end of the command. So not convinced it's worth using `WinRmTool` directly here.

Comment (bostko) [inline winrm4j/src/main/java/io/cloudsoft/winrm4j/winrm/CliClient.java]: Agree I will think for something.

**Comment by neykov** [comment]
Closing, @bostko if you decide to revive it open a new one.

**Pull request closed without merging**